sbi/inference/snpe/snpe_base.py

```diff
@@ -158,9 +159,14 @@ def __call__(
         for round_, num_sims in enumerate(num_sims_per_round):
 
             # Run simulations for the round.
-            theta, x, prior_mask = self._run_simulations(round_, num_sims)
+            theta, x = self._run_simulations(round_, num_sims)
             x_shape = x_shape_from_simulation(x)
 
+            # Curate the data by excluding NaNs. Then append to the data banks.
+            self._append_training_data(
```

Comment:
I am looking for better names - `offline_simulations` or `prior_simulations` come to mind (prior zweideutig...)